README.md

```diff
@@ -242,10 +243,9 @@ See [openhab-js : items](https://openhab.github.io/openhab-js/items.html) for fu
     * .getItem(name, nullIfMissing) ⇒ <code>Item</code>
     * .getItems() ⇒ <code>Array.&lt;Item&gt;</code>
     * .getItemsByTag(...tagNames) ⇒ <code>Array.&lt;Item&gt;</code>
-    * .createItem(itemName, [itemType], [category], [groups], [label], [tags], [giBaseType], [groupFunction], [itemMetadata])

```

Comment:
Why removing it?

Reply:
This function only creates an Item object but does not create a new Item in openHAB.
I think there is no use case where normal scripters should have a use case for that and like in the linked issue, `createItem` only confuses.